src/kbmod/standardizers/butler_standardizer.py

```diff
@@ -141,53 +144,167 @@ def resolveTarget(self, tgt):
+
+        This resolves the majority of the metadata, temporal and
+        spatial information required by KBMOD, which is stored
+        in the attributes and then returned in the standardize
+        mehtods. This prevents having to evaluate queries to the
+        Butler Registry and Datastore repeatedly, and enables the
+        evalution of information that spans multiple datasets without
+        forcing the Butler.get of a larger dataset (f.e. the Exposure
+        object).
+        """
+        self._metadata = {}
+
+        self._metadata["dataId"] = str(self.ref.id)
+        self._metadata["visit"] = self.ref.dataId["visit"]
+        self._metadata["filter"] = self.ref.dataId["band"]
+        self._metadata["detector"] = self.ref.dataId["detector"]
+        self._metadata["collection"] = self.ref.run
+        self._metadata["datasetType"] = self.ref.datasetType.name
+
+        meta_ref = self.ref.makeComponentRef("metadata")
+        meta = self.butler.get(meta_ref)
+        self._metadata["OBSID"] = meta["OBSID"]
+        self._metadata["DTNSANAM"] = meta["DTNSANAM"]  # c4d08927472ksska_ori
+        self._metadata["AIRMASS"] = meta["AIRMASS"]
+        self._metadata["DIMM2SEE"] = meta["DIMM2SEE"]
+        self._metadata["GAINA"] = meta["GAINA"]
+        self._metadata["GAINB"] = meta["GAINB"]
+
+        visit_ref = self.ref.makeComponentRef("visitInfo")
+        visit = self.butler.get(visit_ref)
+        expt = visit.exposureTime
+        mjd_start = visit.date.toAstropy()
+        half_way = mjd_start + (expt / 2) * u.s + 0.5 * u.s
+        self._metadata["exposureTime"] = expt
+
+        # Note the timescales for MJD
+        # f.e. name mjd into mjd_mid - make it obvious it's mdidle of exposure
+        # and add time scale like mjd_mid_utc
+        self._metadata["mjd_start"] = mjd_start.mjd
+        self._metadata["mjd"] = half_way.mjd
+
+        self._metadata["object"] = visit.object
+        self._metadata["pointing_ra"] = visit.boresightRaDec.getRa().asDegrees()
+        self._metadata["pointing_dec"] = visit.boresightRaDec.getDec().asDegrees()
+        self._metadata["instrument"] = visit.getInstrumentLabel()
+        self._metadata["airmass"] = visit.boresightAirmass
+
+        # can I find seeing? gain per amoplifier
+        summary_ref = self.ref.makeComponentRef("summaryStats")
+        summary = self.butler.get(summary_ref)
+        self._metadata["psfSigma"] = summary.psfSigma
+        self._metadata["psfArea"] = summary.psfArea
+        self._metadata["nPsfStar"] = summary.nPsfStar
+        self._metadata["zeroPoint"] = summary.zeroPoint
+        self._metadata["skyBg"] = summary.skyBg
+        self._metadata["skyNoise"] = summary.skyNoise
+        self._metadata["meanVar"] = summary.meanVar
+        self._metadata["astromOffsetMean"] = summary.astromOffsetMean
+        self._metadata["astromOffsetStd"] = summary.astromOffsetStd
+
+        # Will be nan because it's VR filter and task doesn't include it
```

Comment:
Not a silly question. It will be true for DEEP because DEEP was taken in the VR filter (no filter really) and the task that creates summary statistics only works with ugrizy filters. So the effective metrics don't exist because the task never found any data it could run on. So if it was DECam data in DEEP that was in a filter, that would likely have those values is my understanding.